app/views/spaces/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="container socks row-well" id="all_socks">
```

Comment:
Lets discuss layout changes for the site, and relevant custom CSS that will need to be removed.